### PR TITLE
Azure GA docs updates

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by MarkWoulfeNeo4j** [comment]
[Update Azure information for GA](https://trello.com/c/NiUuD0Il/178-update-azure-information-for-ga)

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-aura-230.surge.sh

File: modules/ROOT/pages/platform/security.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 All data stored in Neo4j Aura is encrypted using intra-cluster encryption between the various nodes comprising your database and encrypted at rest using the underlying cloud provider's encryption mechanism.
 
-By default, Google Cloud and AWS encrypt all backup buckets (including the objects stored inside) with either https://cloud.google.com/storage/docs/encryption/default-keys[Google-managed encryption] or https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingServerSideEncryption.html[AWS SSE-S3 encryption].
+By default, all cloud providers encrypt all backup buckets (including the objects stored inside) with either https://cloud.google.com/storage/docs/encryption/default-keys[Google-managed encryption], https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingServerSideEncryption.html[AWS SSE-S3 encryption] or https://learn.microsoft.com/en-us/azure/storage/common/storage-service-encryption[Azure Storage encryption].
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/security.adoc:8]: By "all" we mean Google + Amazon + Microsoft, right?

Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/platform/security.adoc:8]: note that this page follows the cloud providers page that outlines the three offerings, you're very unlikely to land directly on this page with no other context

**Review by nvitucci — APPROVED** [review]
Just a suggestion on the wording, otherwise looks good.

**Pull request merged**